anu_lms.install

```diff
@@ -66,3 +71,41 @@ function anu_lms_update_8001() {
     }
   }
 }
+
+/**
+ * Migrate is_highlight boolean field on image bullet list paragraphs to
+ * default color
+ */
+function anu_lms_update_8002() {
+  /** @var NodeInterface[] $lessons */
+  $lessons = Drupal::entityTypeManager()->getStorage('node')
+    ->loadByProperties(['type' => 'module_lesson']);
+  foreach ($lessons as $lesson) {
+    /** @var EntityReferenceRevisionsItem[] $sections */
+    $sections = $lesson->get('field_module_lesson_content')
+      ->referencedEntities();
+    foreach ($sections as $section) {
+      /** @var Paragraph $contents */
+      $contents = $section->get('field_lesson_section_content')
+        ->referencedEntities();
+      foreach ($contents as $content) {
+        if ($content->getType() == 'lesson_img_list') {
+          /** @var FieldItemList $highlight_field */
+          $highlight_field = $content->get('field_lesson_img_list_highlight');
+          if ($highlight_field->getString() == "1") {
```

Comment:
@treppo what if the value is 0 - do we want to assign "blue" color?

Reply:
in that case it's not highlighted, so it has no color